package.json

```diff
@@ -36,6 +36,7 @@
     "w3c-keyname": "2.2.4"
   },
   "devDependencies": {
-    "process": "^0.11.10"
+    "process": "^0.11.10",
+    "shadow-cljs": "^2.28.20"
```

Comment:
@borkdude is this what triggers all the additions in yarn.lock? Do we need this?

Reply:
Done!